exporters/ironic.go

```diff
@@ -37,18 +38,22 @@ func NewIronicExporter(config *ExporterConfig, logger log.Logger) (*IronicExport
 		}
 	}
 
+	// Remove v1 from ResourceBase
+	config.ClientV2.ResourceBase = config.ClientV2.Endpoint
```

Comment:
This probably needs a note in the code itself as well.

V2 adds this new field ResourceBase. For whatever reason, it adds a `v1` field to the URL, so it sends requests to `/v1/v1`. Doing this fixes the problem.